Frontend/src/components/create-account-page/CreateAccount.tsx

```diff
@@ -1,18 +1,66 @@
+import { useState } from 'react';
 import { ICreateInputs } from '../../assets/utils/types/types';
 import CreateAccountInput from './CreateAccountInput';
 import GoBackButton from './GoBackButton';
 
 const CreateAccount = () => {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [formData, setFormData] = useState<{ [key: string]: string }>({
+    firstname: '',
+    lastname: '',
+    email: '',
+    password: '',
+  });
+
   const inputs = [
     { id: 1, type: 'firstname', text: 'firstname', placeholder: 'first name' },
     { id: 2, type: 'lastname', text: 'lastname', placeholder: 'last name' },
```

Comment:
Dessa har jag lagt här, men bättre om du kan flytta dem och ge ett bättre namn. Lägg den i en models map och importera in. My bad 😄